src/benchmarklib/tpch/tpch_query_generator.cpp

```diff
@@ -52,6 +71,330 @@ std::string TPCHQueryGenerator::get_preparation_queries() const {
 }
 
 std::string TPCHQueryGenerator::build_query(const QueryID query_id) {
+  using namespace std::string_literals;
```

Comment:
Can you doc which of these values are from the TPCH standard and, ideally, from which chapter/paragraph they come from? This would allow for somewhat easy verification that we're standard-conform.

Reply:
Having something like "Generation rules taken from TPCH 2.4.[query].3" in the comment is precisely what I meant - if I were a fresh reader of this, this would help greatly to orient me.